config/locales/en/helpers.yml

```diff
@@ -64,3 +66,14 @@ en:
       steps_case_codefendants_form:
         first_name: First name
         last_name: Last name
+      steps_case_case_type_form:
+        case_type_options:
+          summary_only: Summary only
+          either_way: Either way
+          indictable: Indictable
+          already_cc_trial: Trial already in crown court
```

Comment:
Nitpick but in some places we are titleizing `Crown Court` and in others we are using all lower case `crown court` 😅